.setup/install_worker.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Usage:
+#   install_worker.sh [--no-rpi]
+
+# Read through the flags passed to the script reading them in and setting
+# appropriate bash variables, breaking out of this once we hit something we
+# don't recognize as a flag
+
+export NO_RPI=0
```

Comment:
This should be setup to operate in the same fashion as `install_system.sh` where we should only use `rpi.sh` if they pass the `EXTRA=rpi` environment variable to it. The stuff installed there is not used at all within development, and given our move to Docker, is likely to get phased out altogether.

Reply:
I think I've done this correctly in my new commit using `${@}` to pass on the positional arguments, but I'm not sure.